reconcile/glitchtip_project_alerts/integration.py

```diff
@@ -160,6 +146,44 @@ def fetch_desired_state(
                             ],
                         )
                     )
+
+                elif (
+                    glitchtip_project.jira.escalation_policy
+                    and glitchtip_project.jira.escalation_policy.channels.jira_board
+                ):
+                    # definition via escalation policy
+                    channels = glitchtip_project.jira.escalation_policy.channels
+                    for board in channels.jira_board:
+                        if not integration_is_enabled(
+                            QONTRACT_INTEGRATION, board
+                        ) or not integration_is_enabled(
+                            jira_permissions_validator.QONTRACT_INTEGRATION, board
+                        ):
+                            continue
+                        params["labels"] = alert_labels + (channels.jira_labels or [])
+                        if channels.jira_component:
+                            params["components"] = [channels.jira_component]
+                        if board.issue_type:
+                            params["issue_type"] = board.issue_type
```

Comment:
`params` init outside of `for` loop, if there are multiple boards, later board will use previous board `params` when `if` conditions not met.

Reply:
Now the default value `"Bug"` is always needed for `issue_type`, is it used anywhere else in this repo? If you still need it to be optional in final params, and since the `token` is the only additional field, we can make it shareable like:


```python
                token_params = {"token": gjb_token} if gjb_token else {}
...
                        params = {
                            "labels": alert_labels + (channels.jira_labels or []),
                            "components": [channels.jira_component] if channels.jira_component else [],
                        } | token_params
                        if board.issue_type:
                            params["issue_type"] = board.issue_type
```